kuka_kr270_support/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(kuka_kr270_support)
+
+find_package(catkin REQUIRED)
+
+catkin_package()
+
+if (CATKIN_ENABLE_TESTING)
+  find_package(roslaunch REQUIRED)
+  roslaunch_add_file_check(test/roslaunch_test.xml)
```

```suggestion
  roslaunch_add_file_check(test/roslaunch_test_kr270r2700.xml)
```
there might be other models with a payload of 270kg added later

---

Done in 383d1d9df60931724b02ab95d1238591107c3575